ekncontent/tests/ekncontent/testDictionaryObjectModel.js

```diff
@@ -0,0 +1,44 @@
+const Eknc = imports.gi.EosKnowledgeContent;
+
+const InstanceOfMatcher = imports.tests.InstanceOfMatcher;
+
+describe ('Dictionary Object Model', function () {
```

Comment:
whitespace between `describe` and `(`. There's also multiple instances of whitespace before `function` and `(`, but I'm not sure if that's intentional.